src/libsystemd-network/dhcp-lease-internal.h

```diff
@@ -75,6 +75,8 @@ struct sd_dhcp_lease {
         uint16_t mtu; /* 0 if unset */
 
         char *domainname;
+        char **search_domains;
+        size_t search_domains_count;
```

Comment:
if we maintain such element counters we tend to prefix them with "n_", rather than suffix them with "_count", i.e. `n_search_domains`